english/jumping-jacks/jump.md

```diff
@@ -0,0 +1,3 @@
+You manage to do 30 jumping jacks before tiring. Now you're ready to start your adventure.
+
+[Go back](english/marshmallow.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.